Backend/JourneyLog.DAL/Entities/User.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace JourneyLog.DAL.Entities;
+
+public class User
+{
+    public Guid IdentityId { get; set; }
```

Comment:
I guess plain Id would be better than IdentityId

Reply:
Done